src/widgets/Tabs/PatientSelect.js

```diff
@@ -253,12 +256,16 @@ const PatientSelectComponent = ({
           rowKey={keyExtractor(item)}
           columns={columns}
           onPress={name => {
-            // Only show a spinner when the name doesn't exist in the database, as we need to
-            // send a request to the server to add a name store join.
-            if (UIDatabase.get('Name', name?.id)) {
-              selectPatient(name);
+            if (!name?.isDeceased) {
+              // Only show a spinner when the name doesn't exist in the database, as we need to
+              // send a request to the server to add a name store join.
+              if (UIDatabase.get('Name', name?.id)) {
```

Comment:
:nit same concern as before about nesting and early returns

Reply:
45f396cc5946091857b64a3a527ad97f4456a372